### PR TITLE
describe source NAT fields

### PR DESCRIPTION
**Pull request by DaanHoogland**
to go with [cloudastack#6442](https://github.com/apache/cloudstack/pull/6442)

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

File: source/adminguide/networking/advanced_zone_config.rst [hunk 1]
```diff
@@ -68,6 +68,8 @@ configure the base guest network:
 
    -  **IPv6 DNS**: A set of custom IPv6 DNS that will be used by the guest network. If not provided then IPv6 DNS specified for the zone will be used. Available only when the selected network offering is IPv6 enabled and supports DNS service.
 
+   -  **IPv4 address for the VR in this Network**: The source nat address or primary public network address to use by the guest network. If not provided then a random address from the available pool of addresses wil be used.
```
Comment (harikrishna-patnala) [inline source/adminguide/networking/advanced_zone_config.rst:71]:
```suggestion
   -  **IPv4 address for the VR in this Network**: The source NAT address or primary public network address to use by the guest network. If not provided then a random address from the available pool of addresses wil be used.
```

File: source/adminguide/networking/virtual_private_cloud_config.rst [hunk 2]
```diff
@@ -214,6 +214,7 @@ addresses in the form of a Classless Inter-Domain Routing (CIDR) block.
 
    -  **IPv6 DNS**: A set of custom IPv6 DNS that will be used by this VPC. If not provided then IPv6 DNS specified for the zone will be used. Available only when the selected VPC offering is IPv6 enabled and supports DNS service.
 
+   -  **IPv4 address for the VR in this Network**: The source nat address or primary public network address to use by the guest network. If not provided then a random address from the available pool of addresses wil be used.
```
Comment (harikrishna-patnala) [inline source/adminguide/networking/virtual_private_cloud_config.rst:217]:
```suggestion
   -  **IPv4 address for the VR in this Network**: The source NAT address or primary public network address to use by the guest network. If not provided then a random address from the available pool of addresses wil be used.
```

**Review by harikrishna-patnala — CHANGES_REQUESTED** [review]
minor changes requested, otherwise LGTM

**Review by rohityadavcloud — APPROVED** [review]

File: source/adminguide/networking/virtual_private_cloud_config.rst [hunk 3]
```diff
@@ -214,6 +214,7 @@ addresses in the form of a Classless Inter-Domain Routing (CIDR) block.
 
    -  **IPv6 DNS**: A set of custom IPv6 DNS that will be used by this VPC. If not provided then IPv6 DNS specified for the zone will be used. Available only when the selected VPC offering is IPv6 enabled and supports DNS service.
 
+   -  **IPv4 address for the VR in this Network**: The source NAT address or primary public network address to use by the guest network. If not provided then a random address from the available pool of addresses wil be used.
```
Comment (weizhouapache) [inline source/adminguide/networking/virtual_private_cloud_config.rst:217]: @DaanHoogland 
IPv4 address for the VR in this VPC ?

**Pull request merged**